pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -350,3 +350,75 @@ func CreateObjects(client *client.Client, objects []runtime.Object) {
 	}
 	fmt.Println("\nApplication has been deployed to Kubernetes. You can run 'kubectl get deployment,svc' for details.")
 }
+
+func DeleteObjects(client *client.Client, name string) {
+	err := client.Services(api.NamespaceDefault).Delete(name)
+	if err != nil {
+		logrus.Fatalf("Error: '%v' while deleting service: %s", err, name)
+	}
+	logrus.Infof("Successfully deleted service: %s", name)
+
+	err = client.Deployments(api.NamespaceDefault).Delete(name, nil)
+	if err != nil {
+		logrus.Fatalf("Error: '%v' while deleting deployment: %s", err, name)
+	}
+	logrus.Infof("Successfully deleted deployment: %s", name)
+}
+
+func DeleteAll(client *client.Client) {
+	//delete all svc
+	listOpts := api.ListOptions{}
+	svcs, err := client.Services(api.NamespaceDefault).List(listOpts)
```

Comment:
Flag help says delete everything in all namespaces but here only default namespace is referenced